### DISCUSSION TITLE
How do you add a line break / newline within a Table cell?

### DISCUSSION THREAD

**Discussion by brendand** [gonzalezreal/swift-markdown-ui#216] [Q&A] [answered]
According to the my research, you can insert a line break into a markdown table cell using the HTML `<br>` tag.

```
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title |
| Paragraph   | First paragraph. <br><br> Second paragraph. |
```
But that did not work in MarkdownUI.

Is there a different way to accomplish this in MardownUI?

**Comment by gonzalezreal** [comment]
Hi @brendan,

MarkdownUI does not render inline HTML yet. You can try two spaces at the end of the line and a single new line, which is the standard way of adding a line break in Markdown. I haven't tried that inside a table, though.

**Comment by brendand** [comment]
Hi @gonzalezreal ,

That didn't work inside a table cell.

If rendering some HTML is on your roadmap that'd be great. But for now it's ok. It was a request by one of my beta testers so I thought I'd check to see if there was a way.

**Reply by gonzalezreal** [reply]
Yes, support for some HTML tags is definitely on the roadmap. So I will take into account this feedback. Thanks!

**Reply by gonzalezreal** [reply, marked as answer]
Hey @brendand,

I prepared a pull request to render HTML line breaks:
- https://github.com/gonzalezreal/swift-markdown-ui/pull/221

Let me know if it works for you.

**Reply by brendand** [reply]
Wow! It works perfectly! That was so quick of you. Thank you so much!
